### PR TITLE
v10 Release

### PR DESCRIPTION
**Pull request by joshblack**
# DO NOT MERGE

### CODE REVIEW

**Comment by joshblack** [comment]
cc @connor-leech if you have content feedback 👍

File: README.md [hunk 1]
```diff
@@ -5,16 +5,40 @@
...
-not intended for production.
+Carbon elements is a collection of packages that enable developers to use the
+IBM Design Language in code. We current offer the following packages that you
+can start using today:
+
+| Package name                          | Description                                                                                                                                         |
+| ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`@carbon/colors`](./packages/colors) | Working with colors like Blue 50                                                                                                                    |
+| [`@carbon/grid`](./packages/grid)     | Building layouts using the new 16 column grid system                                                                                                |
+| [`@carbon/icons`](./packages/icons)   | Rconography assets. We also offer support in: [React](./packages/icons-react), [Angular](./packages/icons-angular), and [Vue](./packages/icons-vue) |
```
Comment (aledavila) [inline README.md]: Is this meant to say Iconography ?

Comment (aledavila) [reply README.md]: Line 23

**Comment by jeanservaas** [comment]
@joshblack already slacked about this, just an issue with Layout, i know you're on it!

image

**Pull request merged**